canonical-websites/build.snapcraft.io
# Show spinner on buttons without affecting their width

Fixes #662 

Making sure spinner during loading shows on buttons instead of the text (rather then next to it), not changing its width.

### QA

1. Go to add repos, select some repos, click Add button
2. Spinner should show on Add button in place of the text (not changing buttons width)
3. On my repos open 'Register name' dropdown, Sign in
4. Sign in button should show spinner instead of the text (not changing width)
5. Click 'Register name' button
6. Register button should show spinner without changing buttons width
